### ISSUE TITLE
Allow more clean naming for request/response decorators (optionally)

### ISSUE THREAD

**Issue by maximdanilchenko** [maximdanilchenko/aiohttp-apispec#36]
From this:
```python
@docs(
    tags=["mytag"], 
    summary="Method summary", 
    description="Method description",
)
@use_kwargs(SomeSchema(strict=True))
@marshal_with(AnotherSchema(), 200)
async def index(request):
    return web.json_response({"msg": "hello"})
```
To this:
```python
@docs(
    tags=["mytag"], 
    summary="Method summary", 
    description="Method description",
)
@request(SomeSchema(strict=True))
@response(AnotherSchema(), 200)
async def index(request):
    return web.json_response({"msg": "hello"})
```

**Comment by mirrorrim** [comment]
May be it's better to name `request_schema` and `response_schema`? In this case we will have no conflicts like in example above (`request` decorator and `request` argument).  
If so, I can prepare PR with these changes.

**Comment by maximdanilchenko** [comment]
Yes, `request_schema` and `response_schema` fits better. About PR - it would be great!

**Comment by maximdanilchenko** [comment]
Closed with #40
